### PR TITLE
altered createDATIMExport and created createPAWExport

### PR DESCRIPTION
**Pull request by flopez-bao**
- altered createDATIMExport and created createPAWExport - this is to address divergence of PAW and DATIM exports
- added pertinent tests
- edited packForDATIM function documentation
- related to https://jira.pepfar.net/browse/DP-1198 and datapack import tickets

### CODE REVIEW

File: R/createPAWExport.R [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#'
+#' @param d
+#'
+#' @return A data frame in standardized DHIS2 export format for PAW
+#' @export
+#'
+createPAWExport <- function(d) {
+
+  if (d$info$tool == "Data Pack") {
+    if (d$info$cop_year == 2022) {
```
Comment (JordanBalesBAO) [inline R/createPAWExport.R:12]: Lets nuke the 2022 stuff

**Comment by flopez-bao** [comment]
removed 2022 blocks from datapack calls in `createDATIMExport` and `createPAWExport`. Also added a new function as a concept `createDATAExport` plus test which can act as one function to handle export based on the type.

File: R/createDATIMExport.R [hunk 2]
```diff
@@ -31,9 +22,18 @@ createDATIMExport <- function(d) {
...
+      datim_export <-
+        datim_export %>%
+        dplyr::filter(!dataElement %in% pop_data)
+
     }
   }
 
+  # handle OPU
   if (d$info$tool %in% c("OPU Data Pack", "PSNUxIM")) {
     datim_export <-  d$datim$OPU
```
Comment (JordanBalesBAO) [inline R/createDATIMExport.R:38]: Any ideas on how this will impact OPU's?

Comment (flopez-bao) [reply R/createDATIMExport.R:38]: this was already in the OG code, so there should be no change, i only added the comment

**Review by jason-p-pickering — APPROVED** [review]
Looks good to me!

File: R/createDATAExport.R [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+#' @title createDATAExport
+#'
+#' @param d
+#' @param export_type paw or datim are passed as export types
+#'
+#' @return A data frame in standardized DHIS2 export format for DATIM or PAW
+#' @export
+#'
+#'
+createDATAExport <- function(d, export_type = NULL) {
```
[the remark this answers is not in the document]
Comment (flopez-bao) [reply R/createDATAExport.R]: For now we are using createPAWExport and createDATIMExport so removed this one for now.

**Review by JordanBalesBAO — APPROVED** [review]

**Pull request merged**